{"name": "test_case_p11[无效边界值相加【99.01,0】]", "status": "passed", "parameters": [{"name": "a", "value": "99.01"}, {"name": "b", "value": "0"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1636424028508, "stop": 1636424028508, "uuid": "79a71065-ba90-4cdc-be95-2a74732ad68e", "historyId": "b9a118a7d13bdba09540f7603c579a8a", "testCaseId": "5a7695d5e60876d16c61997094849501", "fullName": "test_calculator.TestCalculator#test_case_p11", "labels": [{"name": "suite", "value": "test_calculator"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "DESKTOP-GRKQKVO"}, {"name": "thread", "value": "2424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calculator"}]}